expressions/equality.md

```diff
@@ -0,0 +1,70 @@
+# Equality in Pony
+
+Pony features two forms of equality: by value and by identity. 
+
+## Identiy equality
+
+Identity equality checks in Pony are done via the `is` keyword. `is` verifies that the two items are the same. Primitives are always equal to 
+
+```pony
+if None is None then
+  // TRUE!
+  // There is only 1 None so the identity is the same
+end
+
+let a = Foo("hi")
+let b = Foo("bye")
+
+if a is b then
+  // NOPE. THIS IS FALSE
+end
+
+let c = a
+if a is c then
+  // YUP! TRUE!
+end
+```
+
+## Value equality
```

Comment:
I often here @sylvanc call this "structural equality" - is that a less ambiguous term to use here?